pallets/parachain-staking/src/runtime_api.rs

```diff
@@ -0,0 +1,42 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2022 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::dispatch::fmt::Debug;
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use sp_runtime::Perquintill;
+
+#[derive(Decode, Encode, TypeInfo, MaxEncodedLen, PartialEq, Eq, Debug)]
+pub struct StakingRates {
+	pub collator_staking_rate: Perquintill,
+	pub collator_reward_rate: Perquintill,
+	pub delegator_staking_rate: Perquintill,
+	pub delegator_reward_rate: Perquintill,
+}
+
+sp_api::decl_runtime_apis! {
+	pub trait ParachainStakingApi<AccountId, Balance>
+	where
+		AccountId:  Eq + PartialEq + Debug + Encode + Decode + Clone,
```

Comment:
I think we can get along with simply requiring `Codec`.

Reply:
At least that is the conclusion I got when working on the public credentials stuff.

Reply:
https://github.com/KILTprotocol/kilt-node/pull/413